app/src/main/res/values-it/strings.xml

```diff
@@ -171,216 +171,205 @@ Halten Sie sich an die Hygiene- und Verhaltensregeln."</string>
+    <string name="meldungen_infected_title">Infezione segnalata</string>
+    <string name="meldungen_infected_text">Grazie ! Le persone informate in via anonima si potranno isolare interrompendo la catena di contagio.</string>
+    <string name="foreground_service_notification_error_network_sync">"Errore caricando i dati "</string>
+    <string name="foreground_service_notification_error_bluetooth_disabled">Bluetooth è disattivato</string>
+    <string name="foreground_service_notification_error_battery_optimization">L\'ottimizzazione della battaeria è attivata e impedisce la scansione in background</string>
 
     <!-- Beruhigender Text für Detail-Seite, falls Meldung eingegangen ist -->
-    <string name="meldungen_hinweis_info_text1">"Dies bedeutet nicht, dass du angesteckt wurdest. Aber es ist wichtig, dass du dich über mögliche Massnahmen informierst."</string>
+    <string name="meldungen_hinweis_info_text1">Questo non significa che sei infetto. Ma è importate che ti informi sulle misure da prendere.</string>
 
     <!-- Info-Text "Was bedeuteted das" für Person, die selbst bemeldet hat -->
-    <string name="meldungen_hinweis_info_text1_infected">"Dies bedeutet nicht, dass jemand angesteckt wurde. Aber es ist wichtig, dass jede Person, der du begegnet bist, sich über angemessene Massnahmen informieren kann."</string>
+    <string name="meldungen_hinweis_info_text1_infected">Questo non significa che altre persone sono state infettate. Ma è importate che tutti quelli con cui hai avuto contatti siano informati, così da poter prendere le misure appropriate.</string>
 
     <!-- Titel der Nachricht, die auf Beta-Status Hinweist -->
-    <string name="preview_warning_title">"Vorschau Version"</string>
+    <string name="preview_warning_title">Versione preliminare</string>
 
     <!-- Text der Nachricht, die auf Beta-Status Hinweist -->
-    <string name="preview_warning_text">"Diese App dient als Vorschau, um die DP-3T Implementation zu zeigen. Es ist kein produktiv verwendbarer COVID-19 Tracer."</string>
+    <string name="preview_warning_text">Questa app è stata creata per dimostrare un implementazione di DP-3T.</string>
+    <string name="onboarding_title_2">Automatica e anonima</string>
+    <string name="onboarding_desc_3">In caso di test positivo pupi informare in forma anonuma tutte le persone che hai incontrato, così che possano prendere le necessarie precauzioni.</string>
+    <string name="exposed_info_support_text">Per qualunque informazione o raccomandazione, potete chiamare</string>
+    <string name="bluetooth_setting_text">Abilitare Bluetooth</string>
+    <string name="bluetooth_setting_tracking_explanation_text1">Utilizzando il BLE  (Bluetooth Low Energy)  l\'app riconosce i contatto con altre persone. Questi incontri vengono registrati dall\'app se due persone si trovano vicine per un certo periodo di tempo e vengono cancellati dopo 21 giorni.</string>
```

Comment:
Utilizzando il BLE  (Bluetooth Low Energy)  dp3t-app riesce a fare una scansione con altri dispositivi che stanno eseguendo l\'app in background. Questi incontri vengono salvati se due persone si trovano vicine per un certo periodo di tempo e vengono cancellati dopo 21 giorni.